controllers/administration.js

```diff
@@ -1211,6 +1211,7 @@ router.get(
 					user,
 					password: authHelper.generateConsentPassword(),
 					referrer: req.header('Referer'),
+					age: CONSENT_WITHOUT_PARENTS_MIN_AGE_YEARS,
```

Comment:
age is littlebit to generic, please a more self speaking name for the attribute

Reply:
age is already used in other places for the same var, so while it may make sense to unify this but is out of scope for this tickt.